modules/PySlacLoopback/manifest.yaml

```diff
@@ -0,0 +1,14 @@
+description: Simple driver for Evse Loopback communication
+provides:
+  main:
+    interface: empty
+    description: Nothing to do here
+requires:
+  ev_slac:
+    interface: ev_slac
+  evse_slac:
+    interface: slac
+metadata:
+  license: https://opensource.org/licenses/Apache-2.0
+  authors:
+    - aw@pionix.de
```

Comment:
I can't remember, but I'm pretty sure I didn't write this module.

Reply:
I think you are the author of this. I only copied this module from the ccc-ev branch to this branch. But I can delete this module of course if we dont need that

Reply:
I just deleted the PySlacLoopback module